src/qt/configurenamedialog.cpp

```diff
@@ -98,4 +112,13 @@ void ConfigureNameDialog::onDataEdited(const QString &name)
 {
     ui->dataSize->setText(tr("%1 / %2").arg(name.size()).arg(MAX_VALUE_LENGTH_UI));
     ui->dataSize->resize(ui->dataSize->fontMetrics().horizontalAdvance(ui->dataSize->text()), ui->dataSize->height());
+
+    std::string data = ui->dataEdit->text().toStdString();
+
+    if(IsValidJSON(data) || data.empty()){
+        ui->labelValidJSON->setText(tr("Valid JSON text."));
+    }else{
```

Comment:
The lack of a space before `else` is visually distracting to me, but I'll let @domob1812 decide what the appropriate style is here.

Reply:
Yes I think there should be spaces around `else`, and also a space after `if` and between `){` at the end of the first line.  But the style in the codebase is a bit mixed up anyway, so I won't block changes for style in Qt code.